e2e/e2e.sh

```diff
@@ -59,7 +59,8 @@ if [ "$1" != "no-index" ]; then
 	pushd "$BINARY_DIR"
 	echo "Building index $INDEX"
 	./IndexBuilderMain -l -i "$INDEX" \
-		-n "$INPUT.nt" \
+	    -F ttl \
```

Comment:
Can you add this to the README and the two quickstart documents as well? Also for the Wikidata Quickstart I think we should then remove the uncompression step. I would still save the compressed Wikidata to disk though, just so it doesn't need to be downloaded again if something goes wrong

Reply:
I reworked the two documents. Can you look at it and review this, I hope I didn't make any mistakes.